Rawl_B502/do_forslointib_ulad_iar_coitchiund_in_so.trig

```diff
@@ -194,14 +194,13 @@
 <#Néill>
     a foaf:Person ;
     irishRel:genName "Néill" ;
-    owl:sameAs <http://example.com/LL/do_fhorshlontib_ulad_co_coitchiunn.trig#Néill>;
-    rel:childOf <#Indiu>.
+    owl:sameAs <http://example.com/LL/do_fhorshlontib_ulad_co_coitchiunn.trig#Néill>.
 
 <#Echach>
     a foaf:Person ;
     irishRel:nomName "Echach" ;
     owl:sameAs <http://example.com/LL/do_fhorshlontib_ulad_co_coitchiunn.trig#Echach>;
-    rel:childOf <#Indiu>.
+    rel:parentOf <#Conaill>, <#Éogain>.
```

Comment:
The text says:

> Indiu ben Néill m. Echach máthair Conaill & Éogain

Indiu is spouseOf Níall and parentOf Conall and Éogan. So you're right to remove the `<#Néill> childOf <#Indiu>` but calling Eochu the parent of Conall and Éogan is incorrect (he is their grandfather).

Reply:
Fixed.